src/api/app/queries/workflow_runs_finder.rb

```diff
@@ -6,42 +6,22 @@ class WorkflowRunsFinder
   }.freeze
 
   def initialize(relation = WorkflowRun.all)
+    @initial_relation = relation
     @relation = relation.includes([:token]).order(created_at: :desc)
   end
 
-  def all
-    @relation.all
-  end
+  def reset
```

Comment:
You would not need to do this if this would not be in a scope/where wrapper class...

Reply:
What is a where wrapper class?

Reply:
`WorkflowRunsFinder` wraps a couple of `WorkflowRun.where...` calls